extras/indexing/src/test/java/org/apache/rya/indexing/accumulo/AccumuloIndexingConfigurationTest.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.rya.indexing.accumulo;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Properties;
+
+import org.apache.accumulo.core.security.Authorizations;
+import org.apache.rya.accumulo.AccumuloRdfConfiguration;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class AccumuloIndexingConfigurationTest {
+
+    @Test
+    public void testBuilder() {
+        String prefix = "rya_";
+        String auth = "U";
+        String visibility = "U";
+        String user = "user";
+        String password = "password";
+        String instance = "instance";
+        String zookeeper = "zookeeper";
+        boolean useMock = true;
+        boolean useComposite = true;
+        boolean usePrefixHash = true;
+        boolean useInference = true;
+        boolean displayPlan = false;
+
+        AccumuloIndexingConfiguration conf = AccumuloIndexingConfiguration.builder()//
+                .auths(auth)//
+                .visibilities(visibility)//
+                .setRyaPrefix(prefix)//
+                .useInference(useInference)//
+                .useCompositeCardinality(useComposite)//
+                .displayQueryPlan(displayPlan)//
+                .setAccumuloInstance(instance)//
+                .setAccumuloPassword(password)//
+                .setAccumuloUser(user)//
+                .setAccumuloZooKeepers(zookeeper)//
+                .useMockAccumulo(useMock)//
+                .useAccumuloPrefixHashing(usePrefixHash)//
+                .setAccumuloFreeTextPredicates("http://pred1", "http://pred2")//
+                .setAccumuloTemporalPredicates("http://pred3", "http://pred4")//
+                .useAccumuloTemporalIndex(true)//
+                .useAccumuloEntityIndex(true)//
+                .useAccumuloFreetextIndex(true)//
+                .pcjUpdaterFluoAppName("fluo")//
+                .useOptimalPcj(true)//
+                .pcjTables("table1", "table2").build();
+
+        Assert.assertEquals(conf.getTablePrefix(), prefix);
```

Comment:
This test could be reduced to a single assertion if you were to implement an equals(Object o) method on the built object. Create the expected configuration object, assert it is equal to the built one.